wesnoth-l.po

```diff
@@ -2231,10 +1909,10 @@ msgstr ""
 "Tímhle způsobem se dopracuju k opravdový osamělosti. Pořád mám Harpera a "
 "jsem za toho chlapce vděčnej, ale on opravdu nemůže rozumět starýmu "
 "chlapovi, jako jsem já. Minulost je minulost, to vím. Ale nemohu si pomoct a "
-"v těhle chvílích myslívám na ségru. Vzpomínám na starý časy, jak mě vždycky "
-"tahala na nějaký malý hloupý skotačení, nějakou malou prácičku za pár "
-"zlatejch, malou cetku nebo jen tak pro zábavu. Samozřejmě jsem nikdy nechtěl "
-"jít. Byl jsem vždycky fakt línej. A pořád jsem, dokonce i teď."
+"v těhle chvílích myslívám na ségru. Vzpomínám, jak mě vždycky tahala na "
```

Comment:
mám to připsat Baldrasovi?
```suggestion
"v těchhle chvílích myslívám na ségru. Vzpomínám, jak mě vždycky tahala na "
```

Reply:
No je to moje chyba, není to úmysl, ale vlastně to tam sedí.